### PR TITLE
Remove local images and all volumes

### PR DESCRIPTION
**Pull request by Augi**

### CODE REVIEW

File: src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ComposeDown extends DefaultTask {
             if (extension.removeContainers) {
                 if (getDockerComposeVersion() >= VersionNumber.parse('1.6.0')) {
                     project.exec { ExecSpec e ->
-                        e.commandLine extension.composeCommand('down')
+                        e.commandLine extension.composeCommand('down', '--rmi', 'local', '--volumes')
```
Comment (alenkacz) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy]: why just local images and not all?

Comment (Augi) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy]: Good point. According to the documentation it does this:
> remove only images that don't have an custom name set by the `image` field

So it actually removes images that was built during `docker-compose up` and doesn't remove images pulled from Registry. So if you use for example `cassandra` from image (which is usual) then the plugin doesn't remove it and  so the image isn't pulled in next runs.

Comment (alenkacz) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy]: hmm I understood it differently, I thought that it is images that are built but tagged as a a specified imagename

If you specify image as well as build, then Compose tags the built image with the tag specified in image

Comment (alenkacz) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy]: but anyway, feel free to merge it, I was just wondering... :)

**Pull request merged**